eidas_node/tests/test_saml.py

```diff
@@ -233,36 +326,44 @@ def test_create_light_response_with_unsupported_sub_status(self):
         expected.status.sub_status_code = None
         self.assertEqual(response.create_light_response(), expected)
 
+    def test_str(self):
+        self.assertEqual(
+            str(SAMLResponse(ElementTree(Element('root')), 'relay')),
+            "relay_state = 'relay', document = <?xml version='1.0' encoding='utf-8' standalone='yes'?>\n<root/>\n")
+        self.assertEqual(str(SAMLResponse(None, None)), 'relay_state = None, document = None')
+
 
-class TestCreateEidasAttribute(SimpleTestCase):
-    def test_create_eidas_attribute_known_attribute(self):
-        root = Element('whatever')
-        elm = create_eidas_attribute(root, 'http://eidas.europa.eu/attributes/naturalperson/CurrentFamilyName', True)
-        self.assertIs(elm, root[0])
-        expected = Element('whatever')
-        SubElement(expected, Q_NAMES['eidas:RequestedAttribute'], {
+class TestCreateAttributeElmAttributes(SimpleTestCase):
+    def test_create_attribute_elm_attributes_known_attribute(self):
+        name = 'http://eidas.europa.eu/attributes/naturalperson/CurrentFamilyName'
+        self.assertEquals(create_attribute_elm_attributes(name, None), {
```

Comment:
`assertEquals` is deprecated.

Reply:
I've fixed this in new `TestCreateAttributeElmAttributes` code and the rest will be addresed in #40.